src/lib/utils/client-route.js

```diff
@@ -11,12 +12,10 @@ exports.register = (server, options, next) =>  {
             notes: 'Add filename (e.g. client.js)',
             description: 'Returns client source code',
             auth: false,
-            plugins: {
-                'hapi-swagger': { nickname: 'get' }
-            },
+            plugins: { 'hapi-swagger': { nickname: 'get' } },
             handler: {
                 directory: {
-                    path: path.join(process.cwd(), 'dist', 'client', 'dist'),
+                    path: clientDistPath,
```

Comment:
pedantic change.  did during debugging because on line 7 i did a `fs.readdir(...)` and console.log of the outputs (to make sure client.js was present).  could trash, but whatevs